ReactSkia/sdk/OnScreenKeyBoard.h

```diff
@@ -0,0 +1,175 @@
+ /*
+ * Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+#ifndef OSK_H
+#define OSK_H
+
+#include <semaphore.h>
+
+#include "include/core/SkCanvas.h"
+#include "include/core/SkColor.h"
+
+#include "rns_shell/compositor/Compositor.h"
+#include "rns_shell/common/Window.h"
+#include "rns_shell/platform/graphics/PlatformDisplay.h"
+#include "rns_shell/platform/graphics/WindowContextFactory.h"
+
+#include "NotificationCenter.h"
+#include "RNSKeyCodeMapping.h"
+
+//Suported KeyBoards
+enum OSKTypes {
+    OSK_DEFAULT_TYPE,
+    OSK_ALPHA_NUMERIC_KB=OSK_DEFAULT_TYPE,
+    OSK_NUMERIC_KB,
+    OSK_TYPE_END
+};
+//Suported Themes
+enum OSKThemes {
+    OSK_DEFAULT_THEME,
+    OSK_DARK_THEME=OSK_DEFAULT_THEME,
```

Comment:
For the enums related to types and themes here , why do we need to have default values as part of the enums ?
shouldn't it be default theme = dark theme and default type = alpha numeric ?

Reply:
keyboardAppearance prop has default value: enum('default', 'light', 'dark') . So to Match it is default Theme has been provided.
Similarly KB Type also has defualt value